.gitignore

```diff
@@ -20,10 +19,12 @@ lib64/
 parts/
 sdist/
 var/
+wheels/
+share/python-wheels/
 *.egg-info/
 .installed.cfg
 *.egg
-.venv*
+MANIFEST
```

Comment:
To remove or leave here?

Reply:
I don't know, not sure what this file is even for ;)